File: aggregator/src/main/java/com/thalesminussi/aggregator/model/station/Wind.java
package com.thalesminussi.aggregator.model.station;

import com.fasterxml.jackson.annotation.JsonIgnoreProperties;

@JsonIgnoreProperties(ignoreUnknown = true)
public class Wind {

    private String speed;

    private String deg;

    public String getSpeed() {
        return speed;
    }

    public void setSpeed(String speed) {
        this.speed = speed;
    }

    public String getDeg() {
        return deg;
    }

    public void setDeg(String deg) {
        this.deg = deg;
    }
}

File: station/src/main/kotlin/com/thalesminussi/station/model/Wind.kt
package com.thalesminussi.station.model

import com.fasterxml.jackson.annotation.JsonIgnoreProperties
import java.io.Serializable

@JsonIgnoreProperties(ignoreUnknown = true)
class Wind : Serializable {

    var speed: Double? = null

    var deg: Double? = null
}